joomla/joomla-cms
# [4.0] [com_templates] Use target from event

### Summary of Changes

Minor code cleanup to use target from event instead of hardcoded variable.

### Testing Instructions

Go to template details.
In Editor tab check that clicking folder icons expand the directory tree.
Click "New File" button. In the modal click on some folder icon. Create a file. Check that the file is created in the correct location.

### Expected result

Works like before.

### Documentation Changes Required

No.
